src/devices/LidarLiteV3/AcquistionModeFlag.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.TimeOfFlight.Models.LidarLiteV3
+{
+    /// <summary>
+    /// Acquisition Mode control
+    /// </summary>
+    [Flags]
+    public enum AcquistionModeFlag
+    {
+        /// <summary>
+        /// 0 - Disable reference process during measurement
```

Comment:
I'm not sure what the 0 and 1 refer to by looking at the description, I'd prefer the descriptions be a bit simplified, i.e.:
`Enables reference process during measurement`

Reply:
will clean this up